src/modules/navigator/mission_block.h

```diff
@@ -65,8 +65,27 @@ class MissionBlock : public NavigatorMode
 	MissionBlock(const MissionBlock &) = delete;
 	MissionBlock &operator=(const MissionBlock &) = delete;
 
+	/**
+	 * Check if the mission item contains a navigation position
+	 *
+	 * @return false if mission item should not be part of the trajectory
+	 */
 	static bool item_contains_position(const mission_item_s &item);
 
+	/**
+	 * Check if the mission item contains a gate condition
+	 *
+	 * @return true if mission item is neither a position nor a command
+	 */
+	static bool item_contains_gate(const mission_item_s &item);
+
+	/**
+	 * Check if the mission item contains a marker
+	 *
+	 * @return true if mission item is neither a position nor a command
```

Comment:
Also here and above your comment is implying something and likely to be wrong in the future. The real truth is probably just the method name.